specification.markdown

```diff
@@ -423,15 +423,15 @@ The `id` attribute is REQUIRED.
 
 #### minOccurs
 
-An integer expressing the minimum number of occurrences of this `EBML Element` within its `Parent Element`. The `minOccurs` value MUST be equal to the `EBMLMinOccurrence` value of the `path`.
+An integer expressing the minimum permitted number of occurrences of this `EBML Element` within its `Parent Element`. The `minOccurs` value MUST be equal to the `EBMLMinOccurrence` value of the `path`.
 
 The `minOccurs` attribute is OPTIONAL. If the `minOccurs` attribute is not present then that `EBML Element` has a `minOccurs` value of 0.
 
 #### maxOccurs
 
-An integer expressing the maximum number of occurrences of this `EBML Element` within its `Parent Element`. The `maxOccurs` value MUST be equal to the `EBMLMaxOccurrence` value of the `path`.
+An integer expressing the maximum permitted number of occurrences of this `EBML Element` within its `Parent Element`. The `maxOccurs` value MUST be equal to the `EBMLMaxOccurrence` value of the `path`.
 
-The `maxOccurs` attribute is OPTIONAL. If the `maxOccurs` attribute is not present then that `EBML Element` has no maximum occurrence, similar to `unbounded` in the XML world.
+The `maxOccurs` attribute is OPTIONAL. If the `maxOccurs` attribute is not present then no further restrictions are imposed on the number of occurences of this `EBML Element` besides the restriction imposed by `minOccurs`. This is similar to `unbounded` in the XML world.
```

Comment:
Referencing `minOccurs` here add more complexity to understand what happens if `maxOccurs` is not present. Saying "there's no restriction on that number except if there is one" doesn't help IMO. I prefer the original text.

Reply:
The rest of the changes I can live with, but I'd rather have this change removed or simplified.